src/server/api.js

```diff
@@ -72,4 +72,25 @@ function submitApplication(req, res) {
   }
 }
 
-module.exports = { submitApplication };
+
+function getApplicationStatus(req, res) {
+  const id = req.params.id;
+  if (id === undefined) {
+    // TODO(all) what should be returned here?
+    res.status(500).json({ error: 'need id' });
+  }
+  const request = {
+    formSubmissionId: id
+  };
+  voaService.getFormSubmissionStatus(request, (err, response) => {
+    if (err) {
+      debug(`voaService response had error ${err}`);
+      // TODO(awong): This may leak server config info on error. Is that a problem?
+      res.status(500).send({ error: err });
```

Comment:
aaand...I think I screwed this up slightly. This should be `.json({ error: err })` and not `.send()`.

The test is to look and see if the response header has `Content-Type: 'application/json'` appropriately set using `.send()`. If not, can we switch all these to .json?